modules/manage/partials/mountable-topics.adoc

```diff
@@ -0,0 +1,211 @@
+  "topics": [
+    {
+      "source_topic": {"ns": "kafka", "topic": "<source-topic-1-name>"}, 
+      "alias": {"ns": "kafka", "topic": "<new-topic-1-name>"}
+    }, 
+    {
+      "source_topic": {"ns": "kafka", "topic": "<source-topic-2-name>"}
+    }, 
+    {
+      "source_topic": {"ns": "kafka", "topic": "source-topic-3-name"}, 
+      "alias": {"ns": "kafka", "topic": "<new-topic-3-name>"}
+    }
+  ]
+}
+```
+
+* `ns` is the topic namespace. This field is optional and only `kafka` is currently supported.
+* To rename a topic in the target cluster, use the optional `alias` object in the request body. In the example, topics 1 and 3 are given new names in the target cluster, while topic 2 retains its original name.
+
+--
+
+======
+
+When the migration is complete, the target cluster handles produce and consume workloads for the topics.
+
+== Cancel a migration
+
+You can cancel a topic migration by running the command:
+
+[tabs]
+======
+rpk::
++
+--
+```
+rpk cluster storage cancel-mount <migration-id>
+```
+--
+
+Admin API::
++
+--
+```
+curl -X POST http://localhost:9644/v1/<migration-id>/?action=cancel
+```
+--
+======
+
+You cannot cancel migrations in the following <<monitor-progress,states>>:
+
+- `planned` (but you may still xref:api:ROOT:admin-api.adoc#delete-/v1/migrations/-id-[delete] a planned migration)
+- `cut_over`
+- `finished`
+- `canceling`
+- `cancelled`
+
+== Monitor progress
+
+[tabs]
+======
+rpk::
++
+--
+To list topic migrations, run the command:
+
+```
+rpk cluster storage list-mount
+```
+--
+
+Admin API::
++
+--
+Issue a GET request to the `/migrations` endpoint to view the state of topic migrations:
+
+```
+curl http://localhost:9644/v1/migrations 
+```
+--
+======
+
+The response returns the IDs and state of existing migrations:
+
+|===
+| Migration state | Outbound migration (unmount) | Inbound migration (mount)
+
+| `planned`
+2+| Redpanda validates the migration definition.
+
+
+| `preparing`
+| Redpanda copies all possible topic data, including topic manifests, to the destination bucket or container in object storage.
+| Redpanda recreates the topics in a disabled state in the target cluster. The cluster allocates partitions but does not add log segments yet. Topic metadata is populated from the topic manifests found in object storage.
+
+| `prepared` 
+| The migration is ready to execute. In this state, the cluster still accepts client reads and writes for the topics.
+| Topics exist in the cluster but clients do not yet have access to consume or produce.
+
+| `executing` 
+| The cluster rejects client reads and writes for the topics. Redpanda uploads any remaining topic data that has not yet been copied to object storage. Uncommitted transactions involving the topic are aborted.
+| The target cluster checks that the topic to be mounted has not already been mounted in any cluster.
+
+| `executed` 
+| All unmounted topic data from the cluster is available in object storage.
```

Comment:
Is there meant to be a description of the mounted topic as well?

Reply:
@asimms41 there's [not a lot](https://github.com/redpanda-data/docs/pull/725#discussion_r1795574128) to describe. How about `The target cluster has verified that the topic has not already been mounted.` ?